packages/client/mutations/AddTeamMutation.ts

```diff
@@ -19,6 +19,9 @@ graphql`
     team {
       id
       name
+      teamMembers {
```

Comment:
Can we spread a fragment here? teamMembers are not used in this file directly, so it is unknown why are they here

Reply:
I like it! Updated 👍